### ISSUE TITLE
Bad Gateway after update to 7.5.2

### ISSUE THREAD

**Issue by jurgenhaas** [alerta/docker-alerta#194]
I've updated to 7.5.2 this morning and got this error when using the Alerta CLI client but also from the clients in the browser:

```
Error: Failed to get config from http://127.0.0.1:8091/api/config. Reason: 502 Server Error: Bad Gateway for url: http://127.0.0.1:8091/api/config
```

Went back to 7.5.1 and all started to work fine again. Sorry I haven't got more details at that time as I had to move on. But maybe this already helps to identify to root of the problem. If not, I'm happy to do more tests and try to find out more.

**Comment by satterly** [comment]
Thanks for reporting. Yes, there is a problem with the build. Surprised that Docker continued to push the new image to DockerHub even though the build pipeline tests failed. Need to fix that.

**Comment by satterly** [comment]
Should be fixed by 519a7f85cd07c9583e5c91737c53be0f0385c49d. Please test.

**Comment by jurgenhaas** [comment]
I can confirm, the issue is fixed.
